scripts/build/autotools/CheckSSL.m4

```diff
@@ -0,0 +1,240 @@
+PHP_ARG_WITH([mongodb-ssl],
+             [whether to enable crypto and TLS],
+             [AS_HELP_STRING([--with-mongodb-ssl=@<:@auto/no/openssl/libressl/darwin@:>@],
+                             [Enable TLS connections and SCRAM-SHA-1 authentication [default=auto]])],
```

Comment:
I think all the help options, should be prefixed with "  MongoDB: ", just as I currently do at https://github.com/xdebug/xdebug/blob/master/config.m4#L7